modules/installation-mirror-repository.adoc

```diff
@@ -148,21 +148,27 @@ the `imageContentSources` data that you require when you install your cluster.
 
 ... Record the entire `imageContentSources` section from the output of the previous
 command. The information about your mirrors is unique to your mirrored repository, and you must add the `imageContentSources` section to the `install-config.yaml` file during installation.
-
-
++
 [NOTE]
 ====
 The image name gets patched to Quay.io during the mirroring process, and the podman images will show Quay.io in the registry on the bootstrap virtual machine.
 ====
 
 . To create the installation program that is based on the content that you
 mirrored, extract it and pin it to the release:
+** If your mirror host does not have internet access, run the following command:
```

Comment:
It looks like Internet should be capitalized here according to the IBM Style Guide, though I see a lot of inconsistencies across out repo. I can create a separate PR to fix incorrect instances.

Reply:
Good call! I just learned about that ISG rule the yesterday; I need to go back through all my pre-yesterday PRs and update :sweat_smile:. Thanks!